### PR TITLE
Feature/editcake

### PR DESCRIPTION
**Pull request by Czarrrna**
I guess this is done.
cc/ @pjar @lsolniczek Please add your comments.

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
+
+.error {
+    border:3px solid red;
+    color:red;
+}
+
+
```
Comment (pjar) [inline app/assets/stylesheets/application.css]: Fix indentation

File: app/controllers/cakes_controller.rb [hunk 2]
```diff
@@ -18,6 +18,21 @@ def create
...
+      @cake = Cake.find(params[:id])
+      if @cake.update(cake_params)
+        flash[:notice] = "Zmiany zostały zapisane."
+        redirect_to root_path
+      else
+        flash[:notice] = "Nie dokonałeś zmian."
+        render "edit"
+      end
+    end
+
```
Comment (pjar) [inline app/controllers/cakes_controller.rb]: Fix indentation

File: app/views/cakes/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="error">
+          <ul>
+          <% @cake.errors[:description].each do |msg| %>
+              <li><%= msg %></li>
+          <% end %>
+          </ul>
+        </div>
+    <% end %>
+    <p><%= form.submit "Wyślij" %></p>
+<% end %>
```
Comment (pjar) [inline app/views/cakes/_form.html.erb]: Fix indentation

File: app/views/cakes/index.html.erb [hunk 4]
```diff
@@ -4,15 +4,8 @@
...
     <th>Przepis</th>
   </tr>
-  <% @cakes.each do |cake| %>
-      <tr>
-        <td><%= cake.name %></td>
-        <td><%= cake.kind %></td>
-        <td><%= cake.description %></td>
-      </tr>
-  <% end %>
+      <%= render partial: "cake", collection: @cakes %>
```
Comment (pjar) [inline app/views/cakes/index.html.erb]: Fix indentation.

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Czarrrna) [inline app/assets/stylesheets/application.css]: tak mi domyślnie ustawia RubyMine...

File: app/views/cakes/_form.html.erb (hunk 3 above)
Comment (Czarrrna) [inline app/views/cakes/_form.html.erb]: done in next PR.

File: app/controllers/cakes_controller.rb (hunk 2 above)
Comment (Czarrrna) [inline app/controllers/cakes_controller.rb]: done in next PR

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Czarrrna) [inline app/assets/stylesheets/application.css]: Done in next PR

**Pull request merged**